torch/csrc/jit/symbolic_script.cpp

```diff
@@ -1289,7 +1310,130 @@ const std::vector<std::string> functions = {
+            return torch.add(self, other, alpha=alpha), backward
+
+        def sub_0(self,
+                  other,
+                  *,
+                  alpha: number = 1.0):
+            self_size = self.size()
+            other_size = other.size()
+            def backward(grad_output):
+                grad_other = (-grad_output * alpha)._grad_sum_to_size(other_size)
+                grad_self = (grad_output)._grad_sum_to_size(self_size)
+                return grad_self, grad_other, None
+            return torch.sub(self, other, alpha=alpha), backward
+
+        def sub_1(self,
+                  other: number,
+                  alpha: number = 1.0):
+            def backward(grad_output):
+                return grad_output, None, None
+            return torch.sub(self, other, alpha=alpha), backward
+        
+        def clamp(self,
+                  min: Optional[number],
+                  max: Optional[number]):
+            def backward(grad_output):
+                # Note: duplication of return is intentional to allow fusion
+                if min is not None and max is not None:
+                    mask = ((self >= float(min)) * (self <= float(max))).type_as(self)
+                    return grad_output * mask, None, None
+                elif min is not None:
+                    mask = (self >= float(min)).type_as(self)
+                    return grad_output * mask, None, None
+                elif max is not None: 
+                    mask = (self <= float(max)).type_as(self)
+                    return grad_output * mask, None, None
+                else: #min is None and max is None
+                    return grad_output, None, None
+
+            return torch.clamp(self, min=min, max=max), backward
+
+        def threshold(self,
+                      threshold: number,
+                      value: number):
+            def backward(grad_output):
+                mask = (self >= threshold).type_as(self)
+                return grad_output * mask, None, None
+            return torch.threshold(self, threshold, value), backward
+
+        def fmod(self,
+                 other: number):
+            def backward(grad_output):
+                return grad_output, None
+            return torch.fmod(self, other), backward
+        
+        def remainder(self,
+                      other: number):
+            def backward(grad_output):
+                return grad_output, None
+            return torch.remainder(self, other), backward
+        
+        def addmm(self, 
+                  mat1, 
+                  mat2, 
+                  *, 
+                  beta: number, 
+                  alpha: number):
+            self_size = self.size()
+            def backward(grad_output):
+                self_grad = (grad_output * beta)._grad_sum_to_size(self_size)
+                mat1_grad = grad_output.mm(mat2.t()) * alpha
+                mat2_grad = mat1.t().mm(grad_output) * alpha
```

Comment:
Doesn't have to go into this PR, but to avoid regressions compared to eager mode, backward should depend on the striding of forward https://github.com/pytorch/pytorch/blob/9467e80097524ef712c2bae5c9bc2588e370034c/tools/autograd/templates/Functions.cpp#L506-L538

Reply:
Great idea; I'll make this a separate PR.